src/components/SlateEditor/plugins/contactBlock/ContactBlockForm.tsx

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { RadioButtonGroup } from '@ndla/ui';
+import { ButtonV2 } from '@ndla/button';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { ContactBlockEmbedData } from '@ndla/types-embed';
+import { FieldProps, Formik } from 'formik';
+import { InputV2, TextAreaV2 } from '@ndla/forms';
+import FormikField from '../../../FormikField';
+import validateFormik, { RulesType } from '../../../formikValidationSchema';
+import InlineImageSearch from '../../../../containers/ConceptPage/components/InlineImageSearch';
+import { TYPE_CONTACT_BLOCK } from './types';
+
+interface ContactBlockFormValues {
+  resource: 'contact-block';
+  description: string;
+  jobTitle: string;
+  name: string;
+  email: string;
+  blobColor?: (typeof blobColors)[number];
+  blob?: (typeof blobTypes)[number];
+  metaImageId?: string;
+}
+
+const rules: RulesType<ContactBlockFormValues> = {
+  jobTitle: {
+    required: true,
+  },
+  description: {
+    required: true,
+  },
+  name: {
+    required: true,
+  },
+  email: {
+    required: true,
+  },
+  metaImageId: {
+    required: true,
+  },
+};
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  gap: ${spacing.small};
+`;
+
+interface Props {
+  initialData?: ContactBlockEmbedData;
+  onSave: (data: ContactBlockEmbedData) => void;
+  onCancel: () => void;
+}
+
+const inputStyle = css`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+const inputContentStyle = css`
+  ${inputStyle};
+  height: 130px;
+`;
```

Comment:
Bruk `styled($COMPONENT)` istedenfor. Da blir det litt lettere å skjønne forskjellen på de to.